eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -161,25 +171,40 @@ def fitJWST(eventlabel, s4_meta=None):
                                 else:
                                     new_file.write(line)
 
-                # Set the intial fitting parameters
-                params = p.Parameters(param_file=meta.fit_par)
-
                 # Subtract off the zeroth time value to avoid floating point precision problems when fitting for t0
                 t_offset = np.floor(meta.bjdtdb[0])
                 t_mjdtdb = meta.bjdtdb - t_offset
                 params.t0.value -= t_offset
 
-                # Get the flux and error measurements for the current channel
-                flux = meta.lcdata[channel,:]
-                flux_err = meta.lcerr[channel,:]
-
-                # Normalize flux and uncertainties to avoid large flux values
-                flux_err = flux_err/ flux.mean()
-                flux = flux / flux.mean()
+                # Temporary normalization to avoid large flux values (FINDME: replace when constant offset is implemented)
+                flux = meta.lcdata[0,:] / np.mean(meta.lcdata[0,:200])
+                flux_err = meta.lcerr[0,:] / np.mean(meta.lcdata[0,:200])
+                for i in np.arange(meta.nspecchan-1):
```

Comment:
That would work as a way to set it up too. Whatever normalization you had in there before wasn't working for me - it was setting the out-of-transit to something slightly larger than 1, so that when I tried to fit just a transit model it was always failing. But this issue is only because we don't have a constant multiplicative offset in the model at all, and once we add that in the weird normalization here won't be needed.

Reply:
We do have a constant term working now - you need to add polynomial into the list of models you're fitting in the ecf and add a c0 term in the fitted parameters ecf.

Reply:
Ah I see, thanks! In that case, I haven't edited the polynomial model yet to allow a shared fit, so for now I'll leave it this way, but I'll keep this in mind for after we figure out the basics of getting the shared format to work with models.